.github/workflows/release.yml

```
@@ -0,0 +1,63 @@
+          asset_name: evm-indexer-v${{ github.event.release.tag_name }}-x86_64-unknown-linux-gnu.tar.gz
+          asset_content_type: application/gzip
+
+  notify:
+    needs: build-and-release
+    runs-on: ubuntu-latest
+    if: always()
+    steps:
+      - name: Notify Success
+        if: ${{ needs.build-and-release.result == 'success' }}
+        run: echo "::notice::Release build and upload completed successfully"
+      
```

yamllint

[error] trailing spaces

(trailing-spaces)